felix/bpf-gpl/jump.h

```diff
@@ -8,15 +8,15 @@
 #include "conntrack.h"
 #include "policy.h"
 
-CALI_MAP(cali_v4_state, 4,
+CALI_MAP(cali_v5_state, 5,
```

Comment:
the v4 was for IPv4 ;-) If we are going to reuse it, we may just remove the v4 and start from version 2 (because 1 is special for historical reasons). We may reconsider in the future whether it was a good idea to merge these.